templates/master.yaml

```diff
@@ -179,198 +205,31 @@ write_files:
       "metadata": {
         "name": "network-policy.net.alpha.kubernetes.io"
       },
-      "description": "fication for a network isolation policy",
+      "description": "Specification for a network isolation policy",
       "versions": [
         {
           "name": "v1alpha1"
         }
       ]
     }
-- path: /srv/fallback-server-dep.yml
```

Comment:
IMO we still need the ingress controllers and their fallback servers inside the guest clusters. So I think we have to keep them. @JosephSalisbury has a better overview about this.